.conda/meta.yaml

```diff
@@ -29,3 +29,5 @@ requirements:
         - cerberus >=1.3.5
         - gitpython
         - jinja2
+        - hpcpy>=0.3.0
```

Comment:
Why `hpcpy>=0.3.0` instead of `hpcpy>=0.5.0` as per `benchcab-dev.yaml`?

Reply:
Why indeed. Fixed.